### DISCUSSION TITLE
About Adding Outreachy Contributions

### DISCUSSION THREAD

**Discussion by mina-atef-00** [LesFruitsDefendus/saskatoon-ng#140] [Q&A] [answered]
Do I just add contributions after PRs are merged?

**Comment by tristanlatr** [comment, marked as answer]
Hello @mina-andajos,

Any open PR count as a contribution, wether we accept it or not. I believe that there is something in the outreachy contribution form to mark status of contributions accepted or not yet accepted. 

Talk to you later,
